mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulHomeTest.kt

```diff
@@ -240,10 +241,67 @@ class StatefulHomeTest {
 
     rule.onNodeWithText(strings.sectionPlay).performClick()
     rule.onNodeWithText(strings.newGame).performClick()
+    rule.onNodeWithText(strings.prepareGamePlayOnline).performClick()
 
     rule.onNodeWithText("user2").assertExists()
     rule.onNodeWithText(strings.prepareGameCancel).performClick()
     rule.onNodeWithText("user2").assertDoesNotExist()
-    rule.onNodeWithText(strings.newGame).assertExists()
+    rule.onNodeWithText(strings.prepareGamePlayOnline).assertExists()
+  }
+
+  @Test
+  fun given_StatefulHome_When_creatingOnlineGameFromUI_opensGameScreenWithCorrectOpponent() =
+      runTest {
+    val auth = emptyAuth()
+    val store = buildStore {
+      collection("users") { document("userId2", ProfileDocument(name = "user2")) }
+    }
+    val authFacade = AuthenticationFacade(auth, store)
+    val social = SocialFacade(auth, store)
+    val chess = ChessFacade(auth, store)
+
+    authFacade.signUpWithEmail("user1@email", "user1", "password")
+    val currentUser = authFacade.currentUser.filterIsInstance<AuthenticatedUser>().first()
+    val user2 =
+        social.profile(uid = "userId2", user = currentUser).filterIsInstance<Profile>().first()
+    currentUser.follow(user2)
+
+    val strings =
+        rule.setContentWithLocalizedStrings {
+          ProvideFacades(authFacade, social, chess) { StatefulHome(currentUser) }
+        }
+
+    rule.onNodeWithText(strings.sectionPlay).performClick()
+    rule.onNodeWithText(strings.newGame).performClick()
+    rule.onNodeWithText(strings.prepareGamePlayOnline).performClick()
+    rule.onNodeWithText("user2").performClick()
+    rule.onNodeWithText(strings.prepareGamePlay).performClick()
+
+    rule.onNodeWithContentDescription(strings.boardContentDescription).assertExists()
+    rule.onNodeWithText("user2").assertExists()
+  }
+
+  @Test
+  fun given_StatefulHome_When_creatingLocalGameFromUI_opensGameScreen() = runTest {
```

Comment:
Missing `then` keyword.

Reply:
Keywords start with a lowercase and between keyword we use camel case

Reply:
Fixed in [2a29349](https://github.com/epfl-SDP/android/pull/226/commits/2a293496a9dde6403f42d2a252ef838212eafbb3)